Dockerfile

```diff
@@ -35,6 +35,10 @@ RUN chmod g=u /etc/passwd
 # also allow adding certificates
 RUN chmod -R g=u /etc/pki/ca-trust
 
+# Add dnf repovar to be used in the RHCOS repos
+RUN mkdir -p /etc/dnf/var
```

Comment:
Super minor: We can drop this and assume it exists. Avoids adding another layer to the image. :)